motr/m0crate/crate_index.c

```diff
@@ -158,8 +159,10 @@
 
 /** Global constants for DIX workload */
 enum {
-	/** Upper limit for max_record_size parameter. */
-	CR_MAX_OF_MAX_RECORD_SIZE	= (1 << 30),
+	/** Upper limit for max_key_size parameter. */
```

Comment:
Mention size if KB in comment, same for value size

Reply:
Fixed.